src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1501,6 +1501,38 @@ private void CheckRecordMemberNames(DiagnosticBag diagnostics)
             {
                 diagnostics.Add(ErrorCode.ERR_CloneDisallowedInRecord, member.Locations[0]);
             }
+
+            var targetEqualsMethod = new SignatureOnlyMethodSymbol(
```

Comment:
These unused variables can now be removed (targetEqualsMethod and targetGetHashCodeMethod)